core/src/main/java/com/linecorp/armeria/common/metric/EventLoopMetrics.java

```diff
@@ -41,7 +41,7 @@
  *     - the total number of IO tasks waiting to be run on event loops</li>
  * </ul>
  **/
-final class EventLoopMetrics implements MeterBinder {
+public final class EventLoopMetrics extends AbstractCloseableMeterBinder {
```

Comment:
Because some additional interfaces or public abstract classes could be set to the super later.

Would better to create a general meter interface instead of exposing an external `MeterBinder` or a special interface such as `CloseableMeterBinder`?

I'm still unsure what is the best for the return type.

Reply:
> Would better to create a general meter interface instead of exposing an external MeterBinder or a special interface such as CloseableMeterBinder?

I see, aren't those interfaces already exposed publicly though?

I was thinking the opposite: Returning `CloseableMeterBinder` could avoid breaking changes.

i.e.
```
CloseableMeterBinder elm = MoreMeterBinders.eventLoopMetrics()
```
to
```
EventLoopMetrics elm = MoreMeterBinders.eventLoopMetrics()
```
doesn't break API compatibility

Reply:
> doesn't break API compatibility

It does compile, but the bytecode signature is different. If a method is compiled with the old version, `NoSuchMethodError` could be raised.

> I also prefer not to expose EventLoopMetrics unless users need to use the methods of EventLoopMetrics.

Understood. Let's use `CloseableMeterBinder`.